src/__tests__/index.ts

```diff
@@ -178,7 +178,7 @@ describe("API client", function() {
 
       it("lists test connections", async function() {
         const connections = await moneyhub.listTestConnections()
-        expect(connections.length).to.be.lessThan(10)
+        expect(connections.length).to.be.lessThan(15)
```

Comment:
Not related to bumping the versions, just increasing the amount of expected banks due to adding the newer test banks that we have.